src/index.html

```diff
@@ -9,10 +9,72 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the class names should follow the BEM methodology. Ensure that the modifiers are used correctly. For instance, `calendar--start-day-sun` and `calendar--month-length-31` appear to be modifiers, but it's not clear what block or element they are modifying. You might need to revise these to follow BEM more accurately.